Ginger/GingerCoreCommon/Repository/BusinessFlowLib/ActivitiesGroup.cs

```diff
@@ -521,5 +521,7 @@ public void CalculateExternalId(IValueExpression ve)
             }
         }
 
+        public Guid BusinessFlowGuid { get; set; }
```

Comment:
BusinessflowGuid can be fetched from parent entity. Is there any specific reason that we need to maintain it again here?

Reply:
Removed used existing filed